static/lottie/ads.json

```diff
@@ -3,8 +3,16 @@
         {
             "id": 1,
             "active": true,
-            "action": "https://landx.id/landx-cuan/",
-            "thumbnail":  "https://landx.id/images/thumbnail_landx_cuan.png"
+            "type": "campaign",
+            "action": "https://landx.id/blog/landx-sewaktu-2021/",
+            "thumbnail":  "https://landx.id/images/campaign_sewaktu2021.png"
+        },
+        {
+            "id": 2,
+            "active": true,
+            "type": "announcement",
+            "title" : "Hati-hati dengan segala jenis penipuan. ",
```

Comment:
@RaflyLesmana3003 be careful ya. ini ada attribute yg berbeda dengan item yg lain semisal title dan description. Pastikan compatible

Reply:
siap mas @glendmaatita , aku kasih type sebagai differentiator nya

Reply:
okay